shacl-compact-syntax/scope.md

```diff
@@ -1,53 +1,53 @@
-# Shacl Compact Syntax 1.2 Scope Document
+# SHACL Compact (SHACL-C) Syntax 1.2 Scope Document
 
 As SHACL needs to be updated to support RDF 1.2, there is an opportunity to promote SHACL-C from a CG to WG specification, whilst:
  - Upgrading the SHACL-C spec to support RDF 1.2 features in alignment with upgrades to the other SHACL specs, and
- - Adding support for additional SHACLC-C features
+ - Adding support for additional SHACL-C features
 
 This document outlines the scope of new features to be supported.
 
 ## Aligning with RDF 1.2
 
 Syntactic additions will be required for SHACL-C to support the following:
 
- - [Support for datasets](https://github.com/w3c/shacl/issues/22)
- - [Support for RDF 1.2 (in particular embedded triples)](https://github.com/w3c/shacl/issues/23)
- - [Support for RDF 1.2 text direction](https://github.com/w3c/shacl/issues/24)
+ - [RDF datasets](https://github.com/w3c/shacl/issues/22)
+ - [RDF 1.2 (in particular embedded triples)](https://github.com/w3c/shacl/issues/23)
```

Comment:
```suggestion
 - [RDF 1.2 (occurrences and triple terms](https://github.com/w3c/shacl/issues/23)
```

Reply:
@afs — Good catch. Needs to lose the open-paren, too. Confirm this is satisfactory?
```suggestion
 - [RDF 1.2 triple terms and occurrences](https://github.com/w3c/shacl/issues/23)
```

Reply:
LGTM